lib/Controller/ScrobblerController.php

```diff
@@ -0,0 +1,89 @@
+<?php declare(strict_types=1);
+
+/**
+ * ownCloud - Music app
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Morris Jobke <hey@morrisjobke.de>
+ * @author Pauli Järvinen <pauli.jarvinen@gmail.com>
+ * @copyright Morris Jobke 2013, 2014
+ * @copyright Pauli Järvinen 2019 - 2025
+ */
```

Comment:
You should add you own name as author and copyright owner when creating new files. I will add mine later if I ever make modifications to the file. When modifying existing files, you may add your name there after the existing names but that's up to you.

I have used such such a logic with years of the copyright that the first year is the creation date and the end year is the time of the latest change.